iwyu_tool.py

```diff
@@ -86,10 +86,21 @@ def clang_formatter(output):
     'clang': clang_formatter
 }
 
+
 def get_output(cwd, command):
     """ Run the given command and return its output as a string. """
+    # https://stackoverflow.com/a/4453495
+    # do not alter the current environment,
```

Comment:
Please reflow this comment to 80 chars, use sentence case (leading uppercase char) and a trailing period. We want comments as full sentences.